api_keys.ipynb

```diff
@@ -0,0 +1,113 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## API Keys\n",
+    "\n",
+    "### Elsevier\n",
+    "\n",
+    "Elsevier API service requires users to get their personal API key and to be\n",
+    "connected to a network that has access to Elsevier publications (usually,\n",
+    "universities enable it).\n",
+    "\n",
+    "Go to [Elsevier developer portal](https://dev.elsevier.com/apikey/manage) and\n",
+    "click Create API Key. Follow the instructions.\n",
+    "\n",
+    "Once the request is sent, a new API key will appear in the Registered API Keys\n",
+    "section. Copy the key (the alpha-numeric combination on the right, under API\n",
+    "Key) and paste it into `ELSEVIER_APIKey` field below.\n",
+    "\n",
+    "Make sure your key is enveloped into quotes, so it has to look something like\n",
+    "this:\n",
+    "\n",
+    "    ELSEVIER_APIKey = \"my1939239key\"\n",
+    "\n",
+    "### Dropbox\n",
+    "We use dropbox to permanently store hits and thus prevent repeating queries for same publications.\n",
+    "\n",
+    "Follow the\n",
+    "[instructions](https://github.com/dropbox/dropbox-sdk-python#creating-an-application)\n",
+    "to create a *Dropbox Application* and to *Obtain an Access Token*. Store it\n",
+    "into `DROPBOX_APIkey` field below.\n",
+    "\n",
+    "    DROPBOX_APIkey = \"my911232key\"\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DROPBOX_APIkey = \"\"\n",
+    "ELSEVIR_APIKey = \"\""
```

Comment:
Perhaps it would be nice to switch the order here. If one is not attentive, it could get the keys mixed up, because of the order in the instructions. Also, while at it, perhaps it would be nice to rename this to ELSEVIER*.

Reply:
Agree, will fix it.